openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -57,10 +57,15 @@
 
 def is_logged_in_cookie_set(request):
     """ Check whether the request has logged in cookies set. """
-    return (
-        settings.EDXMKTG_LOGGED_IN_COOKIE_NAME in request.COOKIES and
-        request.COOKIES[settings.EDXMKTG_LOGGED_IN_COOKIE_NAME]
-    )
+    if settings.FEATURES.get('DISABLE_SET_JWT_COOKIES_FOR_TESTS', False):
```

Comment:
This might warrant a comment. I would wonder why the appropriate jwt cookies aren’t just added in the tests. Actually, is it possible to just add appropriate mocked cookies and avoid this special code?